### ISSUE TITLE
Accessibility issue with terminal on PWD site

### ISSUE THREAD

**Issue by kolldani** [play-with-docker/play-with-docker#448]
PWD would be a great possibility to play with Docker container images, etc. but I am unable to use it via terminal on the PWD site, as it is not accessible using with a screen-reader software. Since I'm blind I use a screen-reader. The problem is that I'm unable to type or paste text (e.g. Docker commands) into the terminal. Screen-reader cannot read anything, as I hadn't written anithing in it. I tested this issue with following screen-readers: JAWS 2021, NVDA, Narrator. 
I tried to use ssh using access information to my instance from site but I got an "Permission denied (publickey).") error. Should I access PWD with ssh?

Thanks in advanc!e!

**Comment by marcosnils** [comment]
Hey @kolldani thank for opening this issue! Since PWD uses https://xtermjs.org/ as a terminar, AFAIK there's an option to enable screenReader mode into the terminal. Regarding copy / paste, you should be able to use ctrl + insert and shift + insert respectively as a copy/paste option. 

I'll make a fix with the screen reader mode on and come back here to see if that works better for you. 

Marcos.

**Comment by marcosnils** [comment]
hey @kolldani! just released a new version of https://labs.play-with-docker.com with accessibility features enabled in the terminal. Since I don't have a screen reader I'd really like your input on how to improve these aspects on the website. Please let me know if it's better now and how we could be improving on this aspect.
